cmd/kube-controller-manager/app/plugins.go

```diff
@@ -45,6 +46,22 @@ import (
 	"github.com/golang/glog"
 )
 
+// ProbeAttachableVolumePlugins collects all volume plugins for the attach/
+// detach controller. VolumeConfiguration is used ot get FlexVolumePluginDir
+// which specifies the directory to search for additional third party volume
+// plugins.
+// The list of plugins is manually compiled. This code and the plugin
```

Comment:
Nope. This PR was large enough as is, didn't want to go refactoring this ugly code and make it larger. Will do that separately (https://github.com/kubernetes/kubernetes/issues/14217).